test/BF16/brgemm-tpp.mlir

```diff
@@ -1,17 +1,7 @@
-// RUN: tpp-opt -transform-dialect-interpreter -one-shot-bufferize="bufferize-function-boundaries allow-return-allocs function-boundary-type-conversion=identity-layout-map" --convert-vnni-to-tpp %s | FileCheck %s
-
-transform.sequence failures(propagate) {
- ^bb0(%arg1: !pdl.operation):
-  %0 = transform.structured.match ops{["linalg.batch_reduce_matmul"]} in %arg1
-    transform.structured.pack %0 { use_vnni=true, blocking_factors = [2] }
-}
+// RUN: tpp-opt -pack-vnni="block-factors=2" -generalize-tensor-pack-unpack -empty-tensor-to-alloc-tensor -one-shot-bufferize="bufferize-function-boundaries allow-return-allocs function-boundary-type-conversion=identity-layout-map" --convert-vnni-to-tpp %s | FileCheck %s
 
 func.func @myfunc(%arg0: tensor<32x4x4xbf16>, %arg1: tensor<32x4x4xbf16>, %arg2: tensor<4x4xbf16>) -> tensor<4x4xbf16>{
-// CHECK: %[[PACK:.+]] = memref.alloc() {alignment = 128 : i64} : memref<32x2x4x2xbf16>
-// CHECK: linalgx.pack %arg1 inner_dims_pos = [1] inner_tiles = [2] into %[[PACK]] : (memref<32x4x4xbf16> memref<32x2x4x2xbf16>) 
-// CHECK: tpp.vnni_brgemm ins(%arg0 : memref<32x4x4xbf16>, %[[PACK]] : memref<32x2x4x2xbf16>) out(%arg2 : memref<4x4xbf16>)
+  // CHECK: tpp.vnni_brgemm ins(%{{.*}} : memref<32x4x4xbf16>, %{{.*}} : memref<32x2x4x2xbf16>) out(%{{.*}} : memref<4x4xbf16>)
```

Comment:
are there no `tensor.pack`s here?

Reply:
It is at memref level there is no pack at that abstraction.

Reply:
Then why do we need `-pack-vnni="block-factors=2" -generalize-tensor-pack-unpack`?

Do we have other VNNI packing tests? We can't simply remove all of them and not replace with anything.

Reply:
@KavithaTipturMadhu do we have?

Reply:
This one's a unit test that checks if linalg.batch_reduce_matmul is lowered correctly. This one needs a pack that must be checked using filecheck.

Reply:
yes, as long as there's an explicit check for pack, its fine.

Reply:
Done in: e158ef01c51ed5036fb40932f1b79d65ca6cb4e1